examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<p>Can we also mention how many circuits required for each step for each method (If that provide a stronger story, otherwise nevermind)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
I will for now not worry about this. Let me explain offline.